### ISSUE TITLE
Where to go next?

### ISSUE THREAD

**Issue by clark-stevenson** [clark-stevenson/paper.d.ts#16]
Hey @StickerFiend 

👍 

So I am not sure of the best way really.  

Ideally I would be happy to transfer the whole thing to you so you can ditch me as the third wheel but I don't really know how to achieve that as not the best with Github. 

You could also just copy the files from here, start your own Repo StickerFiend/paper.d.ts 

Then I could put a big readme and a big dirty comment at the top of my version pointing them to your repo.  This is what  @natelong did a few years back https://github.com/natelong/pixi-ts-def when there was multiple competing declarations and this was before pixi typescript existed. 

If you have experience of using these declarations successfully, maybe even paperjs would let you add a `types`  to their package.json to ship it along with the library.  

options options!

**Comment by natelong** [comment]
I agree that getting the definitions into the upstream project is always the best option, but if that's not possible, it's pretty easy to transfer ownership of a project in GitHub.

@clark-stevenson You should be able to just go [here](https://github.com/clark-stevenson/paper.d.ts/settings) and scroll down to the bottom for "Transfer ownership" in the "Danger Zone" section. The only downside of this route is that anyone previously using your project will need to update their remotes to point to the new location.

Best of luck!
Nate

**Comment by StickerFiend** [comment]
I'll make a pull request on paperjs/paper.js and see if they accept it.

**Comment by willismorse** [comment]
To start with, thanks to everyone who has worked on these TS defs. They've been working great for me.

I may have found an item or two that needs to be added to the def file... is this github repo still the canonical source for the paper.js def project, or has there been progress on integrating this into the larger paper.js project? Just looking to see which is the best repo in which to file an issue/pr.

Thanks,
Will

**Comment by clark-stevenson** [comment]
Apologies for the delay. 

I took a look at this version and the https://www.npmjs.com/package/@types/paper and there really wasn't much out of sync.  I transferred a few optional properties from here to there and I will make a Commit. 

At the time of writing both will be the same, but this one is now Archived, the readme says so and I won't allow any more PR. 

@willismorse if you feel that some things could be improved, I would recommend making a PR with the changes to Definitely Typed. Everyone can install it from `npm install @types/paper --save-dev` which is a normalized method of acquiring. 

I Appreciate everyone's help over the months.  👍
